File: GXpert/GXpert.Web/Modules/Masters/BloomsTaxanomy/BloomsTaxanomyForm.cs
using Serenity.ComponentModel;
using System;

namespace GXpert.Masters.Forms;

[FormScript("Masters.BloomsTaxanomy")]
[BasedOnRow(typeof(BloomsTaxanomyRow), CheckNames = true)]
public class BloomsTaxanomyForm
{
    public string CoginitiveSkill { get; set; }
    public int SortOrder { get; set; }
}

File: GXpert/GXpert.Web/Modules/ServerTypes/Masters/BloomsTaxanomyForm.ts
import { StringEditor, IntegerEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface BloomsTaxanomyForm {
    CoginitiveSkill: StringEditor;
    SortOrder: IntegerEditor;
}

export class BloomsTaxanomyForm extends PrefixedContext {
    static readonly formKey = 'Masters.BloomsTaxanomy';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!BloomsTaxanomyForm.init)  {
            BloomsTaxanomyForm.init = true;

            var w0 = StringEditor;
            var w1 = IntegerEditor;

            initFormType(BloomsTaxanomyForm, [
                'CoginitiveSkill', w0,
                'SortOrder', w1
            ]);
        }
    }
}